pyocd/tools/lists.py

```diff
@@ -89,7 +89,10 @@ def list_boards(name_filter=None):
             'boards' : boards
             }
 
+        # Lowercase target names for comparison
         managed_targets = [dev.part_number.lower() for dev in pack_target.ManagedPacks.get_installed_targets()]
+        builtin_target_names = [target_name.lower() for target_name in BUILTIN_TARGETS]
+        target_names = [target_name.lower() for target_name in TARGET]
```

Comment:
@flit not sure if I've gone too far here: I'm converting the target name from the target list to lower case in addition to the target from the board info. The bug in this case was isolated to the board info list.

Reply:
No, I think this is perfect.